Video Player/test.css

```diff
@@ -0,0 +1,162 @@
+html {
+    box-sizing: border-box;
+  }
+  
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  
+  body {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    background: #534f54;
+    min-height: 100vh;
+    background: linear-gradient(135deg, #534f54 0%,#848185 48%,#161516 100%);
+    background-size: cover;
+    align-items: center;
```

Comment:
align-items ve justify-contex arasindaki fark nedir? Buradan cevapla. Bilmiyorsan arastir.

Reply:
justify-content
--
içeriği birincil eksen boyunca, yani yatay hizalamayı ayarlar. satırları örnek verebiliriz (?)

eğer satır hizasında ise;

- `flex-start` : sola hizala
- `flex-end` : sağa hizala
- `center` : yatay olarak ortala
- `space-between` : tüm genişliğe yatay dağıtarak hizala
- `space-around` : tüm genişlik boyunca yatay olarak eşit şekilde hizala, ancak kenarları boşluklu olacak

align-items
--
içeriği ikincil eksende, yani dikey olarak hizalamayı sağlar. 
-> yön satır ise dikey hizalamayı, sütun ise yatay hizalamayı ayarlar

eğer satır hizasında ise;

- `flex-start` : dikey olarak üste hizala
- `flex-end`: dikey olarak alta hizala
- `center`: dikey olarak ortala
- `stretch`: divin yüksekliğinde olmasına zorla. (sütunlar için güzel çalışır.)